cmd/commands/root.go

```diff
@@ -102,6 +103,11 @@ func GetClientsets(ctx context.Context) *k8sutil.Clientsets {
 		logging.Fatal(err)
 	}
 
+	clientsets.Dynamic, err = dynamic.NewForConfig(clientsets.KubeConfig)
```

Comment:
```suggestion
	clientsets.Dynamic, err = dynamic.NewForConfig(clientsets.KubeConfig)
```

good to see this, I was not aware of this, in the future we can use more of this and minimize the bash command usages. Thanks for using this @Javlopez